src/logcollector/read_win_event_channel.c

```diff
@@ -520,19 +518,7 @@ void send_channel_event(EVT_HANDLE evt, os_channel *channel)
                 channel->evt_log);
         }
         else {
-            avoid_dup = strchr(msg_from_prov, '\r');
-
-            if (avoid_dup){
-                num = avoid_dup - msg_from_prov;
-                memcpy(filtered_msg, msg_from_prov, num);
-                filtered_msg[num] = '\0';
-                cJSON_AddStringToObject(event_json, "Message", filtered_msg);
-            } else {
-                win_format_event_string(msg_from_prov);
-                
-                cJSON_AddStringToObject(event_json, "Message", msg_from_prov);
-            }
-            avoid_dup = '\0';
+            cJSON_AddStringToObject(event_json, "Message", msg_from_prov);
         }
     } else {
         cJSON_AddStringToObject(event_json, "Message", "No message");
```

Comment:
Remove the `else` condition. If the message cannot be read, we should not include it into the event for consistency with the other fields. Same for the decoder:

https://github.com/wazuh/wazuh/blob/80b09587120ab673e3f7982c8aac8e5aac412359/src/analysisd/decoders/winevtchannel.c#L746